generate.go

```diff
@@ -4,7 +4,6 @@
 
 package wf
 
-//go:generate go run generators/gen_guids.go includes/fwpmu.h zguids.go
```

Comment:
Don't remove this, it's still used (although you need a copy of the win10 SDK files manually placed at includes/... for it to work). It's fine to disable it while hacking, but it should stay in place.

Reply:
Oh this wasn't supposed to be part of the pr